charts/redpanda/testdata/ci/01-default-values.yaml.golden

```diff
@@ -654,23 +661,14 @@ spec:
         - name: kafka-default-cert
           secret:
             defaultMode: 272
-            items:
```

Comment:
huh. Do you know where these diffs are stemming from? It seems incorrect for these to be removed. Could they just be out of date?

Reply:
I didn't want transiple the schema, admin and kafka volume definition in the following console deployement:
* https://github.com/redpanda-data/helm-charts/pull/1402/files#diff-0b45ecf3a48bcb79531c1cf2372a743fa93c58bc1ee5193a798e9decd52bf7e4L245
* https://github.com/redpanda-data/helm-charts/pull/1402/files#diff-0b45ecf3a48bcb79531c1cf2372a743fa93c58bc1ee5193a798e9decd52bf7e4L213
* https://github.com/redpanda-data/helm-charts/pull/1402/files#diff-0b45ecf3a48bcb79531c1cf2372a743fa93c58bc1ee5193a798e9decd52bf7e4L185

Regardless of the certification secret reference all will be mounted to console pod.